### ISSUE TITLE
"created" time

### ISSUE THREAD

**Issue by YQisme** [vuejs/vitepress#1891]
### Is your feature request related to a problem? Please describe.

It would be better to add the 'created' time, not just the 'last updated' time.

### Describe the solution you'd like

![image](https://user-images.githubusercontent.com/80308783/216746601-69016237-4d3c-4bf0-bf6e-9c58cd449ed3.png)
![image](https://user-images.githubusercontent.com/80308783/216746610-1061f3a1-b944-48fe-9a0e-a6bfa35de419.png)

### Validations

- [X] Follow our [Code of Conduct](https://vuejs.org/about/coc.html)
- [X] Read the [docs](https://vitepress.vuejs.org).
- [X] Read the [Contributing Guidelines](https://github.com/vuejs/vitepress/blob/main/.github/contributing.md).
- [X] Check that there isn't already an issue that asks for the same feature to avoid creating a duplicate.

**Comment by kiaking** [comment]
Is it possible to get created time of a file? If so, it could be interesting.

**Comment by bluwy** [comment]
One problem (or requirement) with this is that you'd always need to do a full git clone in order to fetch the entire history of the file, otherwise in most CI where a shallow clone is performed, the last created time is also the last updated time.

**Comment by brc-dd** [comment]
> you'd always need to do a full git clone in order to fetch the entire history of the file

Yeah, but that's needed for correct lastUpdated timestamp too. But more importantly, why does one need to know the created time? Also, the UI in the image the author posted, it's very confusing, like you've to look closely to notice that one has a pencil and other has a plus, and what they mean.

#4892 is basically a duplicate of this, but there too the use case is very narrow. The jsonld comment in that thread is also not much useful, like jsonld is not a built-in feature and they probably are using some custom script to inject/extract the info, then why not just spawn git themselves.

**Comment by bluwy** [comment]
> Yeah, but that's needed for correct lastUpdated timestamp too.

Ah you're right. For some reason I thought lastUpdated works in shallow clone. It's already documented not.

**Comment by brc-dd** [comment]
We will revisit this in future if there is any real reason where created at is helpful.
